src/main/scala/edu/ie3/simona/model/participant/WecModel.scala

```diff
@@ -95,7 +98,7 @@ final case class WecModel(
       wecData: WecRelevantData,
   ): Power = {
     val activePower = determinePower(wecData)
-    val pMax = sMax * cosPhiRated
+    val pMax = sMax.toPower * cosPhiRated
```

Comment:
This looks a bit akward to me. I would prefer a toPower function that uses the apparent power and the cosPhi to calculate the power in Watt.